test/format/format.sh

```diff
@@ -437,13 +442,22 @@ format()
 	fi
 
 	cmd="$format_binary -c "$config" -h "$dir" -1 $args quiet=1"
-	verbose "$name: $cmd"
+	echo "$name: $cmd"
 
 	# Disassociate the command from the shell script so we can exit and let the command
 	# continue to run.
 	# Run format in its own session so child processes are in their own process gorups
 	# and we can individually terminate (and clean up) running jobs and their children.
-	nohup setsid $cmd > $log 2>&1 &
+	eval $env_var setsid $cmd > $log 2>&1 &
+
+	# Check for setsid command failed execution, and forcibly quit.
+	# The RUNDIR is not successfully created in this failure type.
+	sleep 1
+	grep -E -i 'setsid: failed to execute' $log > /dev/null && {
+		failure=$(($failure + 1))
+		force_quit=1
+		echo "$name: job in $dir failed to execute"
+	}
```

Comment:
This seems fragile. Do we not get an error return from the call?

Reply:
Unfortunately, the setsid command will return 0 even if there's execution failure of the command, and that's why we have the silent failures initially. See https://jira.mongodb.org/browse/WT-6172?focusedCommentId=3077651&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-3077651

This section of code is trying to catch the execution failure from the log file, to avoid failing in the same pattern in the future.